src/components/binding/binding-panel.js

```diff
@@ -213,13 +187,51 @@ genotet.BindingPanel.prototype.updateTracks = function() {
       this.container.find('#genes #track-' + index).remove();
     }
   }
-  this.data.tracks.forEach(function(track, index) {
-    var ui = this.container.find('#genes #track-' + index);
-    if (!ui.length) {
-      this.addTrack_();
-    }
-    this.selectGenes_[index].val(track.fileName).trigger('change');
-  }, this);
+
+  // Load binding list and update panel tracks.
+  var params = {
+    type: 'list-binding'
+  };
+  $.get(genotet.data.serverURL, params, function(data) {
```

Comment:
The get wrapper in under loader class. If I want to use this wrapper in panel class, I should send a signal to the view and load the binding list in loader, which is what I did before.